integrations/hanlp.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: integration
+name: HanLP
+description: Use HanLP for Chinese text processing with Haystack
+authors:
+  - name: MaChi
+    socials:
+      github: mc112611
+  - name: deepset
+    socials:
+      github: deepset-ai
+      twitter: deepset_ai
+      linkedin: https://www.linkedin.com/company/deepset-ai/
+pypi: https://pypi.org/project/hanlp-haystack/
+repo: https://github.com/deepset-ai/haystack-core-integrations/tree/main/integrations/hanlp
+type: Preprocessor
+report_issue: https://github.com/deepset-ai/haystack-core-integrations/issues
+logo: /logos/hanlp.png
+version: Haystack 2.0
+toc: true
+---
+
+### **Table of Contents**
+
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+  - [Basic Configuration](#basic-configuration)
+  - [Advanced Configuration](#advanced-configuration)
+  - [Sentence Boundary Respect](#sentence-boundary-respect)
+  - [Custom Splitting Functions](#custom-splitting-functions)
+
+## Overview
+
+You can use [HanLP (Han Language Processing)](https://github.com/hankcs/HanLP) in your Haystack pipelines or as a standalone component for Chinese text processing. HanLP is a comprehensive NLP library for Chinese language processing that provides advanced tokenization, sentence segmentation, and other linguistic analysis capabilities.
+
+The integration provides a specialized `ChineseDocumentSplitter` component that understands the unique characteristics of Chinese text, such as the lack of spaces between words and the multi-character nature of Chinese words.
```

Comment:
Add the documentation if we have it already

Reply:
Unfortunately we don't have a docs page yet.

Reply:
Otherwise good point! We can update the integrations page as soon as we have a docs page about the new component.